doc/nrf/device_guides/fem/fem_nrf21540_gpio.rst

```diff
@@ -0,0 +1,59 @@
+.. _ug_radio_fem_nrf21540_gpio:
+
+Enabling GPIO mode support for nRF21540
+#######################################
+
+The `nRF21540`_ device is a range extender that you can use with nRF52 and nRF53 Series devices.
+The nRF21540 GPIO mode implementation of FEM is compatible with the nRF21540 device and implements the 3-pin PA/LNA interface.
+
+The |NCS| provides code that configures FEM based on devicetree (DTS) and Kconfig information using MPSL.
+The FEM hardware description in the application's devicetree file is an essential part of the configuration.
+To enable FEM support, an ``nrf_radio_fem`` node must be present in the application's devicetree file.
+The node can be provided by the devicetree file of the target board, by an overlay file or through the :makevar:`SHIELD` CMake variable.
+See :ref:`zephyr:dt-guide` for more information about the DTS data structure, and :ref:`zephyr:dt_vs_kconfig` for information about differences between DTS and Kconfig.
```

Comment:
Shouldn't we have include for these texts considering the other two documents (Enabling GPIO+SPI mode support for nRF21540 and Enabling support for front-end modules using Simple GPIO interface) also have these text.
We could fix this later as well.

Reply:
I will make sure to include this with edits from @ankuns .